packages/@aws-cdk/pipelines/README.md

```diff
@@ -738,6 +738,12 @@ new pipelines.CodeBuildStep('Synth', {
 });
 ```
 
+For Assets there's only possibility to interact with the commands, not with the whole step. 
+That's why there's always namespace defined in Assets' stage(s) steps so that user can get 
+variables out from those steps. Reference to those steps just need to create manually e.g.
+from synthesized pipeline stack or by hardcoding. E.g. Assets stage's DockerAsset1 step's 
+namespace is `Assets@DockerAsset1`.
```

Comment:
I don't understand this block of text. Why is this under the Codebuild project section? what does "interacting with commands, not with the whole step" mean? This new functionality needs a brief code example, not just text description, of how to use it.

Reply:
Currently PR just add the namespace to generated CloudFormation, and using that needs to be done manually (including manually that variable namespace and variable name). 

So even after PR user needs to
- add manually variable as exported to Asset steps
- export variable inside step
- refer to variable manually when needed to use

It might be possible to make same kind of approach than is used with `CodeBuildStep.exportedVariable`, but not sure how easy it is as Asset stage is generated different way.

Reply:
@comcalvi Changed the place of description in README along the content. Does it now describe better the use case and usage?